src/components/calendar/Calendar.sass

```diff
@@ -0,0 +1,24 @@
+.fc .fc-toolbar-title 
```

Comment:
fc - Do you think this is a good naming ?

Reply:
By fullcalendar docs I need to override properties